msgcntr/messageforums-api/src/bundle/org/sakaiproject/api/app/messagecenter/bundle/Messages.properties

```diff
@@ -900,4 +901,12 @@ topic_associate_label=Use the following rubric to grade this topic
 option_pointsoverride=Adjust individual student scores
 option_hidepoints=Hide point values (feedback only)
 option_studentpreview=Hide Rubric from student
-grading_rubric=Grading Rubric 
\ No newline at end of file
+grading_rubric=Grading Rubric 
+
+prefs_title=Messages
+# Do not remove prefs_description or prefs_title_override, check SAK-21078 for risk and code for reason
+prefs_description=Configure to forward private messages from all your sites to your primary mail. If you want another email, configure this option in "Messages".
```

Comment:
This language is a little awkward. CC: @fostersdesign for review here.

Reply:
@victorGomollon @SedueRey : can you provide a screenshot of where this prefs_description appears. I agree with @bjones86 that the wording needs to be tweaked a bit. My initial suggestions would be to changes `In case you wanted a forward to a different email address, set up this option under 'Messages'.` to `If you want to forward to a different email address, change this option under 'Messages'.`

However, I don't understand where this "under 'Messages' section is located.

Reply:
Attached is a screenshot with the location of the prefs_description, @fostersdesign @bjones86 
Do you see the translation correct now?

![imagen](https://user-images.githubusercontent.com/58029414/74149039-193afe00-4c07-11ea-8c1b-d7e6d64ca7e0.png)

Reply:
Thanks for the screenshot, @victorGomollon. This really helps me visualize the change.

Now that I've seen this, I think we could simplify the description and put more content into the options rather than just "Yes" and "No". Let me think about this overnight and I'll come back with some suggestions tomorrow. Thanks.

Reply:
How about this?

- description: "Configure an alternative messages forwarding email address in the 'Messages' tool."
- option 1: "Do not forward private messages to my email address"
- option 2: "Forward private messages from all my sites to my main email address"

I also think this Messages notifications section should appear further down the page above the "Syllabus" section.

Reply:
Hello,
By default, the order in which the tools are displayed on the menu is the same order in which those tools are loaded.
However, this order can be adjusted to suit any particular needs. This is done via the properties "prefs.tool.order" in "/sakai/config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties" being the "processRegisteredNotificationItems()" method of the "/sakai/user/user-tool-prefs/tool/src/java/org/sakaiproject/user/tool/UserPrefsTool.java" class, the one in charge of it. 
We believe that it is up to each institution to make this customization to their liking, which is why we haven't modified this configuration.
Greetings.